src/main/java/WordsAnalyser.java

```diff
@@ -0,0 +1,94 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.ArrayList;
+
+public class WordsAnalyser {
+    private static final int CHAR_TO_INT_HELPER = 97;
+    private static final int LATIN_ALPHABET_LENGTH = 26;
+    private int longestWord;
+    private int secondLongestWord;
+    private int totalWords;
+    private ArrayList<String>[][] buckets;
+    private ArrayList<String> wordList;
+    private String originalWord;
+
+    public WordsAnalyser() {
+        buckets = new ArrayList[LATIN_ALPHABET_LENGTH][LATIN_ALPHABET_LENGTH];
+        for (int i = 0; i < buckets.length; i++) {
+            for (int j = 0; j < buckets[i].length; j++) {
+                buckets[i][j] = new ArrayList<>();
+            }
+        }
+        wordList = new ArrayList<>();
+    }
+
+    public void readFromUrl(URL file) {
```

Comment:
Try to avoid declaring void as a return type in methods where we are expecting some results.
```suggestion
    public List<String> readFromUrl(URL file) {
```

Reply:
done!